crates/rpc/rpc/src/eth/helpers/state.rs

```diff
@@ -103,10 +100,48 @@ mod tests {
             evm_config,
             None,
             DEFAULT_PROOF_PERMITS,
-        );
+        )
+    }
+
+    #[tokio::test]
+    async fn test_storage() {
+        // === Noop ===
+        let eth_api = noop_eth_api();
+        let address = Address::random();
+        let storage = eth_api.storage_at(address, U256::ZERO.into(), None).await.unwrap();
+        assert_eq!(storage, U256::ZERO.to_be_bytes());
+
+        // === Mock ===
+        let storage_value = StorageValue::from(1337);
+        let storage_key = StorageKey::random();
+        let storage = HashMap::from([(storage_key, storage_value)]);
+
+        let accounts =
+            HashMap::from([(address, ExtendedAccount::new(0, U256::ZERO).extend_storage(storage))]);
+        let eth_api = mock_eth_api(accounts);
 
         let storage_key: U256 = storage_key.into();
         let storage = eth_api.storage_at(address, storage_key.into(), None).await.unwrap();
         assert_eq!(storage, storage_value.to_be_bytes());
     }
+
+    #[tokio::test]
+    async fn test_get_account_missing() {
+        let eth_api = noop_eth_api();
+        let address = Address::random();
+        let account = eth_api.get_account(address, Default::default()).await.unwrap();
+        assert!(account.is_none());
+    }
+
+    #[tokio::test]
+    async fn test_get_account_empty() {
+        let address = Address::random();
+        let accounts = HashMap::from([(address, ExtendedAccount::new(0, U256::ZERO))]);
+        let eth_api = mock_eth_api(accounts);
+
+        let account = eth_api.get_account(address, Default::default()).await.unwrap();
+        let expected_account =
+            alloy_consensus::Account { code_hash: KECCAK_EMPTY, ..Default::default() };
```

Comment:
I believe this is available from the reth-rpc-types crate so we don't need the alloy-consensus dep here